LongWarOfTheChosen/Src/WotCWeaponsAndArmor/Classes/X2Item_Schematics.uc

```diff
@@ -0,0 +1,388 @@
+// This is an Unreal Script
+//This class will mostly be used to make it so the default schematics are not buildable.
+class X2Item_Schematics extends X2Item;
+
+static function array<X2DataTemplate> CreateTemplates()
+{
+	local array<X2DataTemplate> Schematics;
+
+	Schematics.AddItem(CreateTemplate_VektorRifle_Magnetic_Schematic());
+	Schematics.AddItem(CreateTemplate_VektorRifle_Beam_Schematic());
+
+	Schematics.AddItem(CreateTemplate_Bullpup_Magnetic_Schematic());
+	Schematics.AddItem(CreateTemplate_Bullpup_Beam_Schematic());
+
+	Schematics.AddItem(CreateTemplate_WristBlade_Magnetic_Schematic());
+	Schematics.AddItem(CreateTemplate_WristBlade_Beam_Schematic());
+
+	Schematics.AddItem(CreateTemplate_ShardGauntlet_Magnetic_Schematic());
+	Schematics.AddItem(CreateTemplate_ShardGauntlet_Beam_Schematic());
+
+	Schematics.AddItem(CreateTemplate_Sidearm_Magnetic_Schematic());
+	Schematics.AddItem(CreateTemplate_Sidearm_Beam_Schematic());
+
+	return Schematics;

```

Comment:
If you want to remove the schematics from the game, just remove them from [this list](https://github.com/long-war-2/lwotc/blob/master/LongWarOfTheChosen/Config/XComLW_Overhaul.ini#L326) in `XComGameLW_Overhaul.ini`.

Reply:
Ok cool, was not aware that was there. I'll remove them from that file then and get rid of the schematics class then.

Reply:
Removed the schematics from LW_Overhaul.ini so now the schematic class is not needed to hide them.